
int8 result 		# calibration result

uint8 CALIBRATION_OK = 0
uint8 CALIBRATION_ACTIVE = 2
uint8 CALIBRATION_CANCELLED = 3
uint8 CALIBRATION_FAILED = 4
